<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrievePortalDataWidget</class>
 <widget class="QWidget" name="RetrievePortalDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
  <layout class="QGridLayout" name="subgridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="manifestGroupBox">
     <layout class="QFormLayout" name="manifestFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSearch">
        <property name="text">
         <string>Search file:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDatasetID">
        <property name="text">
         <string>Dataset ID:  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDatasetID"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableViewSearchResult"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonDownload">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonExportVTK">
     <property name="text">
      <string>Export VTK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButtonAnalyse">
     <property name="text">
      <string>Analyse</string>
     </property>
    </widget>
   </item>
  </layout>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
